src/plone/api/content.py

```diff
@@ -666,3 +669,29 @@ def find(context=None, depth=None, unrestricted=False, **kwargs):
         return catalog.unrestrictedSearchResults(**query)
     else:
         return catalog(**query)
+
+
+@required_parameters("name")
+def get_vocabulary(name=None, context=None):
```

Comment:
Please complement the docstring with a link to the example, etc.. See `get_view` how it could look like.